aptos-move/aptos-vm/src/sharded_block_executor/global_executor.rs

```diff
@@ -62,8 +65,10 @@ impl<S: StateView + Sync + Send + 'static> GlobalExecutor<S> {
             BlockExecutorConfig {
                 local: BlockExecutorLocalConfig {
                     concurrency_level: self.concurrency_level,
-                    allow_fallback: true,
+                    allow_sequential_block_fallback: true,
```

Comment:
Can we have something like fn `aptos_prod_block_executor_config` and have a builder pattern for it defined somewhere so we do not need to set/copy all these and it is also clear what are the prod settings? And for profiling, something along the lines of `fn with_block_stm_profiling(mut self) -> Self`.

Reply:
builder pattern sounds good, but I'd hold on until we do some changes that might require further customization, so we can construct the building properly?

Reply:
Sounds good!